packages/perseus/src/widgets/numeric-input/utils.ts

```diff
@@ -42,8 +46,16 @@ export const generateExamples = (
     // Generate a list of the unique answer forms.
     const uniqueForms = getUniqueAnswerForms(answerForms);
 
+    // Sort them by the order they appear in the `formExamples` list.
+    const formExampleKeys = Object.keys(NumericExampleStrings);
+    const sortedForms = uniqueForms.sort((a, b) => {
```

Comment:
So originally we were sorting `answerForms` when deriving the data, but:

1. I think that's a UI concern, not a data concern
2. I didn't want to have to move `NumericExampleStrings` into `perseus-core`

So I moved the sorting logic and added a test.